src/Command/Autoscaling/AutoscalingSettingsSetCommand.php

```diff
@@ -0,0 +1,776 @@
+<?php
+
+namespace Platformsh\Cli\Command\Autoscaling;
+
+use Platformsh\Cli\Command\CommandBase;
+use Platformsh\Client\Exception\EnvironmentStateException;
+use Platformsh\Client\Model\AutoscalingSettings;
+use Platformsh\Client\Model\Deployment\Service;
+use Platformsh\Client\Model\Deployment\WebApp;
+use Platformsh\Client\Model\Deployment\Worker;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class AutoscalingSettingsSetCommand extends CommandBase
+{
+    protected function configure()
+    {
+        $this->setName('autoscaling:set')
+            ->setDescription('Set the autoscaling configuration of apps or workers in an environment')
+            ->addOption('service', '-s', InputOption::VALUE_REQUIRED, 'Name of the app or worker to configure autoscaling for')
+            ->addOption('metric', null, InputOption::VALUE_REQUIRED, 'Name of the metric to use for triggering autoscaling')
+            ->addOption('enabled', null, InputOption::VALUE_OPTIONAL, 'Enable autoscaling based on the given metric')
+            ->addOption('threshold-up', null, InputOption::VALUE_OPTIONAL, 'Threshold over which service will be scaled up')
+            ->addOption('threshold-down', null, InputOption::VALUE_OPTIONAL, 'Threshold under which service will be scaled down')
+            ->addOption('duration-up', null, InputOption::VALUE_OPTIONAL, 'Duration over which metric is evaluated against threshold for scaling up')
+            ->addOption('duration-down', null, InputOption::VALUE_OPTIONAL, 'Duration over which metric is evaluated against threshold for scaling down')
+            ->addOption('cooldown-up', null, InputOption::VALUE_OPTIONAL, 'Duration to wait before attempting to further scale up after a scaling event')
+            ->addOption('cooldown-down', null, InputOption::VALUE_OPTIONAL, 'Duration to wait before attempting to further scale down after a scaling event')
+            ->addOption('instances-min', null, InputOption::VALUE_OPTIONAL, 'Minimum number of instances that will be scaled down to')
+            ->addOption('instances-max', null, InputOption::VALUE_OPTIONAL, 'Maximum number of instances that will be scaled up to')
```

Comment:
I think these wouldn't make sense to specify without a value - meaning they should use VALUE_REQUIRED. The exception is `--enabled` which I guess could mean "true" without a value.